src/estimators/bloom_filters.py

```diff
@@ -396,6 +411,9 @@ def __call__(self, sketch_list):
       return [FirstMomentEstimator._estimate_cardinality_exp(union)]
     if self._method == FirstMomentEstimator.METHOD_UNIFORM:
       return [FirstMomentEstimator._estimate_cardinality_uniform(union)]
+    if self._method == FirstMomentEstimator.METHOD_GEO:
```

Comment:
This class is getting crowded. There is a pretty good case for breaking it up into a subclass per method. Can you add a comment about this somewhere, perhaps around line 286? Maybe the next person who adds a method will take this on.

Reply:
Yes, agree, I'll add comments and update as a follow up